docs/permissions.md

```diff
@@ -29,6 +29,11 @@ Additionally, users with the **ProjectCoordinator** role can:
 
 ## Releasing Outputs
 
+!!! note
+    Releasing of outputs to the Jobs site is currently being rolled out.
+    Setting up a user is documented [here](https://bennettinstitute-team-manual.pages.dev/tech-team/releasing-outputs/) and the new process is documented [here](https://docs.google.com/document/d/1PSzMoCFov2olJpxrFGT1DccWUqGDA67mUvPIyIhOcew/edit#heading=h.aokkzsfuxo4n).
```

Comment:
Maybe avoid "here" links if possible?

Reply:
Are they considered not ideal? Not sure what to replace them with, any suggestions would be greatly appreciated!

Reply:
Maybe something like:

> See the [team manual]() for instructions on setting up a user. The new release process is described separately in [Google Docs](https://docs.google.com/document/d/1PSzMoCFov2olJpxrFGT1DccWUqGDA67mUvPIyIhOcew/edit#heading=h.aokkzsfuxo4n).

NB: I don't know if there is a definitive singular noun equivalent to "Google Docs": "Google Doc"?

***

As for motivation, this is very much opinion-based over evidence, but suggestions in [this Stack Exchange question](https://ux.stackexchange.com/questions/12100/why-shouldnt-we-use-words-such-as-here-and-this-in-textlinks), that "here" links:

* are less helpful for screen reader users skimming links; I found a post on [WebAIM](https://webaim.org/techniques/hypertext/#link_to_link) that backs this up.
* are slightly misguiding in drawing the reader's eye to text with little useful meaning
* contain an extra word that is usually redundant

Reply:
Thank you on both fronts!  Updated text as suggested and rebased in.

Reply:
Another reason is for SEO (not that it matters in this case, but a good habit to get into)